pkg/bucketeer/api/api.go

```diff
@@ -67,7 +116,7 @@ func (c *client) sendHTTPRequest(url string, body interface{}) ([]byte, int, err
 	req.Header.Add(authorizationKey, c.apiKey)
 	req.Header.Add("Content-Type", "application/json")
 	client := &http.Client{
-		Timeout: time.Second * 10,
+		Timeout: 60 * time.Second,
```

Comment:
Other SDKs use 30 seconds as the default, but I'm changing it to 60 seconds because it sometimes gets large data. It should be fine for a server-to-server request.